MAUI/LinearGauge/bar-pointer.md

```diff
@@ -0,0 +1,307 @@
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+			Position = GaugeElementPosition.Outside,
+			Offset = 5
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Customize linear gauge bar pointer offset](images/bar-pointer/bar_pointer_offset.PNG)
+
+## Change the color of bar pointer 
+
+The color of the bar pointer can be changed by the [`Fill`]() property. The following code sample demonstrates the same. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Value="50" Fill="Red"/>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+			Fill = new SolidColorBrush(Colors.Red),
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Apply color to bar pointer](images/bar-pointer/bar_color.PNG)
+
+## Apply gradient
+
+The gradient can be applied by using the [`GradientStops`]() property of bar pointer. The following code sample demonstrates how to apply a gradient to the bar pointer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Position="Outside" Offset="5" 
+                                      CornerStyle="BothCurve" Value="70">
+                        <gauge:BarPointer.GradientStops>
+                            <gauge:GaugeGradientStop Value="0" Color="Green"/>
+                            <gauge:GaugeGradientStop Value="35" Color="#ff0074E3"/>
+                        </gauge:BarPointer.GradientStops>
+                    </gauge:BarPointer>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		ObservableCollection<GaugeGradientStop> gradientStops = new ObservableCollection<GaugeGradientStop>();
+		gradientStops.Add(new GaugeGradientStop() { Value = 0, Color = Colors.Green });
+		gradientStops.Add(new GaugeGradientStop() { Value = 35, Color = Color.FromArgb("#ff0074E3") });
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 70,
+			Position = GaugeElementPosition.Outside,
+			Offset = 5,
+			CornerStyle = CornerStyle.BothCurve,
+			GradientStops = gradientStops
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Apply radial gradient to bar pointer](images/bar-pointer/gradient_bar.PNG)
+
+## Child support
+
+By using [`Child`]() property of bar pointer, you can provide a child for bar pointer. The child content is used to add any UI content like text or images, inside the bar pointer to improve readability.
```

Comment:
By using the [`Child`]() property of bar pointer, you can provide a child for the bar pointer. To improve readability, the child content is used to add any UI content, such as text or images, inside the bar pointer.